exercises/isbn-verifier/src/test/java/IsbnVerifierTest.java

```diff
@@ -0,0 +1,91 @@
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class IsbnVerifierTest {
+    private IsbnVerifier isbnVerifier;
+
+    @Before
+    public void setUp() {
+        isbnVerifier = new IsbnVerifier();
+    }
+
+    @Test
+    public void validIsbnNumber() {
+        Assert.assertEquals(true, isbnVerifier.isValid("3-598-21508-8"));
```

Comment:
This still needs to be changed to `assertTrue` :) And you could do a static import so that you don't have to do `Assert.assertTrue`:
```
import static org.junit.Assert.assertTrue;

assertTrue(something, somethingElse)
```
:)